README.org

```diff
@@ -1,17 +1,33 @@
+* Eventually Matrix
+
+  [[https://en.wikipedia.org/wiki/Eventual_consistency][Eventually]] Matrix is a
+  repository of knowledge about the Matrix ecosystem maintained by the community
+  for the community. It is the "bleeding edge" counterpart of the official
+  Matrix.org repositories in terms of documentation, SDK and libraries.
+
 * Purpose                                              :matrix:eventually_matrix:
 
-  Eventually Matrix is a repository of knowledge about the Matrix ecosystem maintained
-  by the community for the community. In broad strokes, this repository exists for
-  the following purposes in roughly the order they're listed.
+  In broad strokes, this repository exists for the following purposes in roughly the
+  order they're listed.
 
+  - Allow Matrix community to easily document the current behavior of implementations
+    in a specification format when such behaviour deviates from the upstream spec.
+    - Speed up the process of adoption and validation of new features/endpoints.
+    - Give a more bleeding-edge reference document that Matrix community can use to
+      prepare for stable releases of the official specification.
+    - Avoid duplication of work and effort in reverse-engineering deviation from
+      outdated specification.
   - Allow Matrix community to easily document pieces of the de-facto Matrix spec
     which are either undocumented in the official spec or are unclear.
     - To facilitate this documentation occurring as quickly as possible, this kind
       of documentation is not required to be in any specific format. Document the
-      spec however you like (.md, .doc, .txt, whatever).
+      spec however you like (.md, .doc, .txt, whatever). Within forked repositories,
+      please try to match your style to upstream to ease future merging.
     - To the same end, documentation isn't required to be perfect or thorough to
       be accepted; it must simply be an improvement on current documentation of
       the Matrix spec.
+  - Provide a QA/draft environment for the Matrix community to improve on the various
+    pieces of work and specification, before attempting being merge with upstream.
   - Provide a location for the Matrix community to comment on and help with
     in-progress spec changes.
   - Provide a central place for the Matrix community to propose and debate spec
```

Comment:
I see this as equivalent to bullet point you added above. Merge them or choose one or the other I'd say.

Reply:
Indeed, will remove this one.